apps/files/js/mainfileinfodetailview.js

```diff
@@ -163,7 +163,7 @@
 					timestamp: this.model.get('mtime'),
 					date: OC.Util.relativeModifiedDate(this.model.get('mtime')),
 					hasFavoriteAction: hasFavoriteAction,
-					starAltText: isFavorite ? t('files', 'Favorited') : t('files', 'Favorite'),
+					starAltText: isFavorite ? t('files', 'Favoured') : t('files', 'Favorite'),
```

Comment:
```suggestion
					starAltText: isFavorite ? t('files', 'Favoured') : t('files', 'Favour'),
```
But might need a translators hint then to make clear it's a "verb"?
https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#provide-context-hints-for-translators